.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 1.5.0
+current_version = 1.5.1
```

Comment:
The version bump will definitely be more than a hotfix :) Not sure what Peter thinks but I'd say this is 2.0.0 worthy.

Reply:
Yes, I agree! However, I knew someone would suggest the best version. I think 2.0.0 is better.

Reply:
I made it 2.0.0